### DISCUSSION TITLE
Transaction stuck

### DISCUSSION THREAD

**Discussion by bscaine** [blockscout/blockscout#9650] [Support]
Hi, I have a transaction pending for 10 hours already from MaiaDao (https://v3.maiadao.io/) 0xb46d449db5cd1bf5c62f5080335fd3b6b48d1201063b5ad7f1487e85123009c1
All attempts of canceling or speeding it up failed.
Tried again and now I got 2 transactions stuck. 0x0c6eedf6d0d74c62db601a6392e338eff8668dee97dbb0f1adec6c7b3e6a54b2
Same error on both: Error: (Awaiting internal transactions for reason)
Any help? Thanks

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
